README.md

```diff
@@ -10,11 +10,17 @@ Currently, the platform offers developers the option to build HIP and ROCm from
 
 ## Table of Contents
 
-- [Installation From Source](#installation-from-source)
-- [Configuration](#configuration)
-- [Usage](#usage)
-- [Tests](#tests)
-- [Development Manuals](#development-manuals)
+- [TheRock](#therock)
+  - [Description](#description)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I would drop the table of contents from the table of contents ;)

Or keep this focused on only the essentials. The full list of headings is already available in github: 
![image](https://github.com/user-attachments/assets/013c1504-a6a9-4c71-bff1-3edc8a19390f)